### DISCUSSION TITLE
Can I customize the comments or prompts of the code?

### DISCUSSION THREAD

**Discussion by moxisuki** [eslint/eslint#14849] [Q&A]
We have a project that loads JavaScript as a plug-in. This project provides many custom APIs, such as: mc.runcmd('test');
"mc" is a custom object, runcmd is its function

How to add annotations or prompt messages to such custom functions or objects in a project to facilitate the identification of lower-level developers? Especially for the explanation of the parameters and return values, like this, but I hope it is a custom implementation, which will help our plug-in developers to be familiar with our API, which is very important for my project.
![image](https://user-images.githubusercontent.com/59305689/127353836-99ac151d-3b9c-4d3f-ad38-ebe74cede07e.png)

**Comment by nzakas** [comment]
This is not part of ESLint.

**Reply by moxisuki** [reply]
Is this the case? I’m really sorry, I’m using ESLint for the first time, can you give me some suggestions?

**Reply by nzakas** [reply]
It’s part of VS Code. You’d need to look at their docs

**Reply by moxisuki** [reply]
oh!,i see
